python/social_bot/tasks.py

```diff
@@ -166,6 +172,9 @@ def __init__(self,
         self._random_goal = random_goal
         self._distraction_list = distraction_list
         self._object_list = distraction_list
+        self._move_goal_during_episode = move_goal_during_episode
+        self._success_without_angle_requirement = success_without_angle_requirement
```

Comment:
seems self._success_without_angle_requirement is not used.

Reply:
@ruizhaogit, probably missed that part of the change?

Reply:
The missing part is added in new commits.